pkg/tracer/tracer_linux.go

```diff
@@ -5,12 +5,15 @@ package tracer
 import (
 	"bytes"
 	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
```

Comment:
I am not too familiar with the internal workings of reporting metrics. But I don't think our agent anywhere uses prometheus for that. Why are we using it here?

I expected the metric sending to StackState to be done by the stackstate-agent and not by this library.